scripts/common/client.py

```
@@ -0,0 +1,1087 @@
+        resp.raise_for_status()
+        return resp.json()
+
+
+class InteractiveAPI:
+    """Interactive API namespace — yields InteractiveSession via context manager."""
+
+    def __init__(self, client: httpx.Client, ui_server: str):
+        self._client = client
+        self._ui = ui_server
+
+    def session(self, response_windows: bool = False) -> InteractiveSession:
```

Ruff (0.15.0)

[warning] Unused method argument: `response_windows`

(ARG002)